include/deal.II/matrix_free/fe_remote_evaluation.h

```diff
@@ -434,6 +436,65 @@ class FERemoteEvaluationCommunicator : public Subscriptor
 
 
 
+namespace Utilities
+{
+  /**
+   * A factory function for the FERemoteEvaluationCommunicator in the case of
+   * point-to-point interpolation.
+   *
+   * @param[in] matrix_free MatrixFree object that is used to distribute
+   * quadrature points at non-matching faces.
+   * @param[in] non_matching_faces_marked_vertices A vector of boundary face IDs
+   * that relate to non-matching interfaces. Each boundary face ID is
+   * accompanied by a lambda that specifies, where remote points can be found.
+   * @param[in] dof_no DoFHandler number in @p matrix_free.
+   * @param[in] quad_no Quadrature number in @p matrix_free.
+   * @param[in] tolerance Tolerance to find remote points.
+   */
+  template <int dim,
+            typename Number,
+            typename VectorizedArrayType = VectorizedArray<Number>>
+  std::unique_ptr<FERemoteEvaluationCommunicator<dim>>
+  setup_remote_communicator_faces_point_to_point_interpolation(
+    const MatrixFree<dim, Number, VectorizedArrayType> &matrix_free,
+    const std::vector<
+      std::pair<types::boundary_id, std::function<std::vector<bool>()>>>
+                      &non_matching_faces_marked_vertices,
+    const unsigned int dof_no    = 0,
+    const unsigned int quad_no   = 0,
+    const double       tolerance = 1e-9);
+
+  /**
+   * A factory function for the FERemoteEvaluationCommunicator in the case of
+   * Nitsche-type mortaring.
+   *
+   * @param[in] dof_handler Used DoFHandler object.
+   * @param[in] mapping Used Mapping object.
+   * @param[in] non_matching_faces_marked_vertices A vector of boundary face IDs
+   * that relate to non-matching interfaces. Each boundary face ID is
+   * accompanied by a lambda that specifies, where remote points can be found.
```

Comment:
"lambda that specifies, where remote points can be found" -> what does this mean?